core/string/print_string.cpp

```diff
@@ -296,6 +296,13 @@ bool is_print_verbose_enabled() {
 	return OS::get_singleton()->is_stdout_verbose();
 }
 
-String stringify_variants(const Variant &p_var) {
-	return p_var.operator String();
+String stringify_variants(const Span<Variant> &p_vars) {
+	String result;
+	for (const Variant &v : p_vars) {
+		if (!result.is_empty()) {
+			result += " ";
+		}
+		result += v.operator String();
+	}
```

Comment:
This changes behavior. To reproduce previous behavior, I would do this:

```suggestion
	if (p_vars.is_empty() {
		return String();
	}
	String result = String(p_vars[0]);
	for (const Variant &v : Span(p_vars.ptr() + 1, p_vars.size() - 1)) {
		result += " ";
		result += v.operator String();
	}
```